pkg/apis/flowcontrol/types.go

```diff
@@ -296,6 +298,17 @@ type FlowSchemaStatus struct {
 	// +listMapKey=type
 	// +optional
 	Conditions []FlowSchemaCondition
+
+	// `controllerSemVer` holds the semantic version of the controller
+	// that set the status.  A controller of lesser semantic version
+	// is not allowed to subsequently update the status.
+	ControllerSemVer string
```

Comment:
this is fairly unusual... this means a faulty controller cannot be rolled back?

Reply:
Dang.  That's a fatal flaw.
We considered introducing an elected leader to make the one true setting, but did not agree that this is a good idea either.
See #90541 for an alternative approach: just let disagreeing kube-apiservers fight it out, but not so fast that it makes a big stink.